jax/numpy/lax_numpy.py

```diff
@@ -4319,7 +4332,7 @@ def _operator_round(number, ndigits=None):
                     "nonzero", "searchsorted", "round"]
 _diff_methods = ["clip", "conj", "conjugate", "cumprod", "cumsum",
                  "diagonal", "dot", "max", "mean", "min", "prod", "ptp",
-                 "ravel", "repeat", "sort", "squeeze", "std", "sum",
+                 "ravel", "repeat", "sort", "sort_complex", "squeeze", "std", "sum",
```

Comment:
This is a list of functions that also are `np.ndarray` methods; I don't think `sort_complex` falls in this category, so it can be removed from this list.